src/api/style.js

```diff
@@ -172,7 +172,7 @@ export default class Style {
 
     _changed() {
         if (this._changeCallback) {
-            this._changeCallback();
+            this._changeCallback(this);
```

Comment:
why?

Reply:
Because [as you said](https://github.com/CartoDB/renderer-prototype/pull/186#discussion_r175479354)  the changeCallback receives a style argument ;)

Reply:
I see, you changed the behavior, previously it wasn't that way